homeassistant/components/media_player/squeezebox.py

```diff
@@ -303,3 +304,62 @@ def turn_on(self):
         """Turn the media player on."""
         self._lms.query(self._id, 'power', '1')
         self.update_ha_state()
+
+    def play_media(self, media_type, media_id, **kwargs):
+        """
+        Send the play_media command to the media player.
+
+        If ATTR_MEDIA_ENQUEUE is True, add `media_id` to the current playlist.
+        """
+        if kwargs.get(ATTR_MEDIA_ENQUEUE):
+            self.add_uri_to_playlist(media_id)
+        else:
+            self.play_uri(media_id)
```

Comment:
add_uri_to_playlist and play_uri is private functions, can you rename them to start with an underscore? `_play_uri`

Even better would be to remove them completely and just determine the `play` or `add` parameter

```python
enque = 'add' if kwargs.get(ATTR_MEDIA_ENQUEUE) else 'play'
self._lms.query(self._id, 'playlist', enque, media_id)
```

And then move all docs to a homeassistant.io PR